ui/src/app/pages/homepage/recent-resources.tsx

```diff
@@ -77,7 +76,7 @@ interface Props {
   workspaces: WorkspaceResponse[];
 }
 
-const RecentResources = fp.flow(withCdrVersions())((props: Props) => {
```

Comment:
this component needs `cdrVersionListResponse` in the Props - does it get set elsewhere?

Reply:
nope I just forgot to put it back